multi-arch-builders/coreos-ppc64le-builder.bu

```diff
@@ -6,10 +6,6 @@
 #
```

Comment:
I guess we need to delete the "Merge in the builder-common.ign Ignition file" line above (and should probably also remove it from the `multi-arch-builders/coreos-aarch64-builder.bu`) file too, which should have been done when that file was changed originally.

Reply:
Yes, `multi-arch-builders/coreos-aarch64-builder.bu` is already updated, just making the same here

Reply:
and also line 3 from `multi-arch-builders/coreos-aarch64-builder.bu` if you don't mind